scipy/stats/tests/test_morestats.py

```diff
@@ -1987,6 +1987,98 @@ def test_circfuncs_unit8(self):
         assert_allclose(stats.circstd(x, high=180), 20.91551378, rtol=1e-7)
 
 
+class TestQuartileCoeffOfDispersion(object):
+
+    def test_simple_flattened(self):
+        arr = [2, 4, 6, 8, 10, 12, 14, 16]
+
+        # Q1 = 4, Q2 = 12 -> dispersion = (12-4)/(12+4) = 8/16 = 0.5
+        assert_equal(stats.quartile_coeff_dispersion(arr), 0.5)
+
+        arr = [1.6, 2.1, 2.3, 2.4, 2.6, 2.9, 2.98, 3]
+
+        # Q1 = 2.1, Q2 = 2.9 -> dispersion = (2.9-2.1)/(2.9+2.1) = 0.8/5 ~= 0.16
+        assert_almost_equal(stats.quartile_coeff_dispersion(arr), 0.16)
```

Comment:
The documentation for `assert_almost_equal` states:
![image](https://user-images.githubusercontent.com/6570539/106830188-6b5c0700-6642-11eb-8ad3-a6b979d7fb70.png)

Old code uses these old functions, but I'd suggest `assert_allclose` if it does what you want it to.